backend/lib/mail.ts

```diff
@@ -0,0 +1,35 @@
+const nodemailer = require('nodemailer');
+
+async function mail() {
+    let testAccount = await nodemailer.createTestAccount();
+    const now = new Date();
+    const months = ["Styczeń", "Luty", "Marzec", "Kwiecień", "Maj", "Czerwiec", "Lipiec", "Sierpień", "Wrzesień", "Październik", "Listopad", "Grudzień"];
```

Comment:
funkcja do wysyłki maili powinna być jak najbardziej uniwersalna. Np. Jak będziesz potrzebować wysłać maila z inna treścią to tej metody nie użyjesz.